src/test/compound/Compound.t.sol

```diff
@@ -0,0 +1,223 @@
+        0xf650C3d88D12dB855b8bf7D11Be6C55A4e07dCC9, // cUSDT
+        0xccF4429DB6322D5C611ee964527D42E5d685DD6a, // cWBTC2
+        0x80a2AE356fc9ef4305676f7a3E2Ed04e12C33946, // cYFI
+        0xB3319f5D18Bc0D84dD1b4825Dcde5d5f7266d407 // cZRX
+    ];
+
+    function setUp() public {
+        defiBridgeProxy = new DefiBridgeProxy();
+        rollupProcessor = new RollupProcessor(address(defiBridgeProxy));
+        compoundBridge = new CompoundBridge(address(rollupProcessor));
+    }
+
+    function testETHDepositAndWithdrawal(uint256 depositAmount) public {
+        vm.assume(depositAmount > 1e9 && depositAmount < 2**96);
+        vm.deal(address(rollupProcessor), depositAmount);
+
+        AztecTypes.AztecAsset memory empty;
+        AztecTypes.AztecAsset memory depositInputAssetA = AztecTypes.AztecAsset({
+            id: 1,
+            erc20Address: address(0x0000000000000000000000000000000000000000),
+            assetType: AztecTypes.AztecAssetType.ETH
+        });
+        AztecTypes.AztecAsset memory depositOutputAssetA = AztecTypes.AztecAsset({
+            id: 2,
+            erc20Address: cETH,
+            assetType: AztecTypes.AztecAssetType.ERC20
+        });
+
+        // cETH minting
+        (uint256 outputValueA, , ) = rollupProcessor.convert(
+            address(compoundBridge),
+            depositInputAssetA,
+            empty,
+            depositOutputAssetA,
+            empty,
+            depositAmount,
+            1,
+            0
+        );
+
+        assertGt(outputValueA, 0, 'cETH received is zero');
+
+        uint256 redeemAmount = outputValueA;
+        AztecTypes.AztecAsset memory redeemInputAssetA = depositOutputAssetA;
+        AztecTypes.AztecAsset memory redeemOutputAssetA = depositInputAssetA;
+
+        // withdrawing ETH (cETH burning)
+        (outputValueA, , ) = rollupProcessor.convert(
+            address(compoundBridge),
+            redeemInputAssetA,
+            empty,
+            redeemOutputAssetA,
+            empty,
+            redeemAmount,
+            1,
+            1
+        );
+
+        // ETH withdrawn should be approximately equal to ETH deposited
+        // --> the amounts are not the same due to rounding errors in Compound
+        assertLt(
+            depositAmount - outputValueA,
+            1e10,
+            'amount of ETH withdrawn is not similar to the amount of ETH deposited'
+        );
+    }
+
+    function testERC20DepositAndWithdrawal(uint256 depositAmount) public {
+        // Note: if Foundry implements parametrized tests remove this for loop,
+        // stop calling setup() from _depositAndWithdrawERC20 and use the native
+        // functionality
+
+        // For the tests to pass a return value of cTokens has to be >0.
+        // Since the ratio of token/cToken is very skewed plenty of the times
+        // I set relatively high minimum value.
+        vm.assume(depositAmount > 1e14 && depositAmount < 2**96);
```

Comment:
You can use an `uint96` to get rid of the last part of the assume (`testERC20DepositAndWithdrawal(uint96 depositAmount)`. Otherwise, use a `bound` function, believe forge-std got one that will make the value sit inside the range.